build.properties

```diff
@@ -13,4 +13,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-version=14.14.1
+version=14.15.0
```

Comment:
The minor version should change when a new feature is added. In this case, we are just rearranging some json files and fixing a few CLDR generation problems along the way, so I think that is a point release only. Please set this back to 14.14.1